rim/forms.py

```diff
@@ -1,13 +1,17 @@
 from django import forms
-from rim.models import Group, Equipment
+from rim.models import Group, Equipment, Client
 
 class GroupForm(forms.ModelForm):
     class Meta:
         model = Group
         exclude =[]
 
 class EquipmentForm(forms.ModelForm):
-
     class Meta:
         model = Equipment
         exclude = ['latest_checkout']
+
+class ClientForm(forms.ModelForm):
```

Comment:
ClientForm is unnecessary since we don't really use its functionality (you can basically revert this entire file to what it was before). Also, be sure to delete any reference to it in views.py